Core/models.py

```diff
@@ -217,3 +218,22 @@ def get_absolute_url(self):
     def __str__(self):
         return "{} | {}".format(self.title, self.author)
 
+
+class Email_Content(models.Model):
```

Comment:
class name should be in UpperCamelCase not Upper_Camel_Case_Mixed_With_Snake_Case
```suggestion
class EmailContent(models.Model):
```